MANIFEST.in

```diff
@@ -3,3 +3,5 @@ include doc/source/*
 include doc/examples/*.py
 include LICENSE.txt
 include README.rst
+include versioneer.py
+include satpy/version.py
```

Comment:
Isn't this included automatically because it is a python file inside the package?